src/Microsoft.SqlTools.Credentials/Microsoft.SqlTools.Credentials.csproj

```diff
@@ -19,7 +19,7 @@
   </ItemGroup>
   <ItemGroup>
 	<PackageReference Include="System.Data.SqlClient" Version="4.4.0-preview1-25305-02" />
-	<PackageReference Include="Microsoft.SqlServer.Smo" Version="140.2.3" />
+	<PackageReference Include="Microsoft.SqlServer.Smo" Version="140.2.4" />
 	<PackageReference Include="Microsoft.SqlServer.Management.SqlScriptPublishModel" Version="140.2.0" />
```

Comment:
oops, it looks like I didn't bump this instance of the SqlScriptPublishModel package reference.  It doesn't appear to have been properly revev'ed for at least a couple update iterations